plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -364,6 +364,5 @@ class ScaleBarImpl : ScaleBar, View {
     internal const val MSG_RENDER_CONTINUOUS = 1
     internal const val DEFAULT_MAPVIEW_WIDTH = 0F
     internal const val DEFAULT_PIXEL_RATIO = 1.0F
-    internal const val INTERNAL_PADDING_DP = 10
```

Comment:
Do you remember why this was introduced?

Reply:
We can add tests while pixel match test is ready.